public/app/features/browse-dashboards/components/utils.ts

```diff
@@ -9,3 +10,12 @@ export function makeRowID(baseId: string, item: DashboardViewItemWithUIItems) {
 export function isSharedWithMe(uid: string) {
   return uid === config.sharedWithMeFolderUID;
 }
+
+// Append orgId to the folder URL
+export function getFolderURL(url: string) {
+  const { orgId } = contextSrv.user;
+  if (orgId) {
+    return `${url}?orgId=${orgId}`;
+  }
+  return url;
+}
```

Comment:
I had more in mind that this function should construct the URL from bare minimum arguments. Something like 

```ts
function getFolderURL(uid: string) {
  // TODO: make sure subpathis honored
  const url = `/dashboards/f/${uid}`;
  if (orgId) {
    let url += `?orgId=${orgId}`;
  }
  return url;
}
```

Reply:
Seems like an overkill if we already have constructed URLs and just need to append orgId?

Reply:
Ok, that is constructed on frontend, so we can just move that logic into this function.